helpers.py

```diff
@@ -167,24 +212,29 @@ def find_app_by_short_name(short_name, pbclient):
         return response[0]
     except exceptions.ConnectionError:
         raise
-    except:
-        format_error("pbclient.find_app", response)
+    except ProjectNotFound:
+        raise
```

Comment:
what happens here with exceptions that are neither ConnectionError nor ProjectNotFound?